pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.29</version>
+    <version>2.30</version>
```

Comment:
Given that this will cause https://youtrack.jetbrains.com/issue/IDEA-175538 for people running intelliJ, I would favour keeping this on 2.29 for now